src/render.jl

```diff
@@ -24,13 +24,27 @@ end
 
 ## 2 arg form render for comparing
 function render(mode::BeforeAfter, reference, actual)
+    if showable(MIME("image/png"), actual)
```

Comment:
This `if` statement is always true if `ImageShow` is loaded, but what I actually wanted to check is if there's an available display for it.

Do you know how could I achieve it? @oxinabox

Reply:
I think `displayable` instread of `showable`